comicScraper.py

```diff
@@ -55,10 +69,10 @@ def folderCBZPacker(path, issuename="Complete"):
         shutil.make_archive(comicTitle + "-" + issuename, 'zip', path + "/" + issuename)
     os.rename(comicTitle + "-" + issuename + ".zip", comicTitle + "-" + issuename + ".cbz")
 
-# TODO: this could cause an issues when downloading a single issue
 def getIssueName(issueLink, startURL):
     # first get the issue name/number. remove the start url, trim the leading /, and everything after the ?
-    issueName = issueLink.replace(startURL, "", 1)[1:].split("?",1)[0]
+    issueName = issueLink.replace(startURL, "", 1)[0:].split("?",1)[0]
```

Comment:
Issue titles were fucking up and losing the first character, then fixing it caused other issues to show up with a forward slash in front. Not sure if that was me fucking it or some sort of unusual issues